app/models/metadata.rb

```diff
@@ -58,4 +58,33 @@ def self.metadata_to_json
     Metadata.all.order(id: :asc).to_json
   end
 
+  def self.to_csv
+    csv = ''
+    metadata_columns = Metadata.new.attributes.keys
+    metadata_columns.delete("created_at")
+    metadata_columns.delete("updated_at")
+
+    metadata_columns.map! { |e|
+      e.gsub("_", " ").capitalize
+    }
+
+    csv << metadata_columns.join(',')
+    csv << "\r\n"
```

Comment:
You probably don't need the `\r`. Same below

Reply:
Thanks I have fixed this.